content/events/2023-new-york-city/propose.md

```diff
@@ -0,0 +1,8 @@
++++
+Title = "Propose"
+Type = "event"
+Description = "Propose a talk for devopsdays New York City 2023"
++++
+  {{< cfp_dates >}}
+
+<strong>How to submit a proposal:</strong> {{< event_link url-key="cfp_link" text="Papercall" >}}
```

Comment:
We need to confirm if we're using papercall or something else and subscribe to it. We've struggled with papercall and i dont think its been updated in years. I'm not seeing many events use it at this point.